SPECS/ig/ig.spec

```diff
@@ -52,5 +61,5 @@ rm $ig_file
 
 %changelog
 * Tue Mar 14 2023 Francis Laniel <flaniel@linux.microsoft.com> - 0.25.0-1
```

Comment:
One small thing: please bump `Release`. We can no longer skip doing that as we published (or will soon publish) packages to 3.0 preview.